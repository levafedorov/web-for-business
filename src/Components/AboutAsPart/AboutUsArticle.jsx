import React from 'react';




export default function AboutUsArticle(props){
     


  

    return <article>
        <h2>About us</h2>
        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.
             Doloremque aliquam qui saepe
         quaerat quos, unde sapiente aliquid nobis architecto praesentium?</p>
    </article>
}