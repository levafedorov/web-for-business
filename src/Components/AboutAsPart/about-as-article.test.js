import React from 'react';
import {shallow} from 'enzyme';
import AboutUsArticle from './AboutUsArticle';



describe("AboutUsArticle component", () => {
    test("Should AboutUsArticle component render", () => {
        const component = shallow(<AboutUsArticle/>);
        expect(component).toMatchSnapshot();
    });
});