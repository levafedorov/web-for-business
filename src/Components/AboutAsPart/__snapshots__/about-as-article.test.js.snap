// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUsArticle component Should AboutUsArticle component render 1`] = `
<article>
  <h2>
    About us
  </h2>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloremque aliquam qui saepe quaerat quos, unde sapiente aliquid nobis architecto praesentium?
  </p>
</article>
`;
